### ISSUE TITLE
Add slash commands support

### ISSUE THREAD

**Issue by jaARke** [UF-ACE/stock-prediction#15]
The Discord API allows us to provide users with hints when they start to type one of our bot's commands (see: [Slash Commands](https://support.discord.com/hc/en-us/articles/1500000368501-Slash-Commands-FAQ). We should add support for these hints to make it easier for users to interact with the bot. There are several online resources that can help here, but some of the more helpful are linked below:

[How To Add Autocompletion To Your Discord Slash Commands](https://www.youtube.com/watch?v=zSzFHxOkCfo)
[Cogs and Slash Commands [Discord.py v2.0+]](https://www.youtube.com/watch?v=U0Us5NHG-nY)

**Comment by jaARke** [comment]
Initial support has been added to the sentiment cog. In the future, we **should add a slash command for each successful implementation of a regular command.** Slash commands will allow users to interface with our retrieval and analysis functions without having to submit a public message in a Discord server. Slash commands also provide users with argument hints, which might be more useful than using `$help`.

In other words, for each bit of functionality we add from here on out, we should make it available via a regular AND a slash command.
